frontline-functions-quickstart/README.md

```diff
@@ -0,0 +1,58 @@
+# Frontline Functions Quickstart
+
+Creates 1 function for each callback needed to setup Frontline features. 
+
+This is intended to be used as the Integration Service of a Frontline instance.
+
+## Pre-requisites
+
+### Environment variables
+
+This project requires some environment variables to be set. A file named `.env` is used to store the values for those environment variables. To keep your tokens and secrets secure, make sure to not commit the `.env` file in git. When setting up the project with `twilio serverless:init ...` the Twilio CLI will create a `.gitignore` file that excludes `.env` from the version history.
+
+In your `.env` file, set the following values:
+
+| Variable | Description | Required |
+| :------- | :---------- | :------- |
+| WORKER_USERNAME | SSO username of agent to be assigned to Example Customer | Yes |
+| PHONE_NUMBER_FOR_CUSTOMER_1 | Phone number to be used on the Example Customer 1 | Yes |
+| NAME_FOR_CUSTOMER_1 | Display name for Example Customer 1 | Yes |
+| PHONE_NUMBER_FOR_CUSTOMER_2 | Phone number to be used on the Example Customer 2 | No |
+| NAME_FOR_CUSTOMER_2 | Display name for Example Customer 2 | No |
+| TWILIO_PHONE_NUMBER | A Twilio phone number to be used as the Sender | Yes |
```

Comment:
The `TWILIO_WHATSAPP_NUMBER` is missing form this list

Reply:
done